Cargo.lock

```diff
@@ -3118,12 +3118,9 @@ dependencies = [
 
 [[package]]
 name = "log"
-version = "0.4.17"
+version = "0.4.21"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "abb12e687cfb44aa40f41fc3978ef76448f9b6038cad6aef4259d3c095a2382e"
-dependencies = [
- "cfg-if",
-]
+checksum = "90ed8c1e510134f979dbc4f070f87d4313098b704861a105fe34231c70a3901c"
```

Comment:
This shouldn't be part of the PR since it's already in `albatross`. Did you miss to rebase `albatross`?

Reply:
I rebased with github. I will fix it

Reply:
Fixed